app/gauge.tsx

```diff
@@ -2,10 +2,14 @@ export const Gauge = ({
   value,
   size = "small",
   showValue = true,
+  color = "text-[hsla(131,41%,46%,1)]",
+  bgcolor = "text-[#333]",
 }: {
   value: number;
   size: "small" | "medium" | "large";
   showValue: boolean;
+  color: String;
+  bgcolor: String;
```

Comment:
Can you make these two props optional, otherwise the builds fail as the demo expects these props currently.